cmake.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+language: cmake
+contributors:
+    - ["Bruno Alano", "https://github.com/brunoalano"]
+filename: CMake
```

Comment:
The filename could be `CMakeLists.txt` -- only the code parts are included in the generated file.